src/Backend/Modules/Pages/Layout/Templates/Add.html.twig

```diff
@@ -73,7 +73,7 @@
               {# Do not change the ID! #}
               <p class="current-template">{{ 'lbl.Template'|trans|ucfirst }}:
                 <span id="tabTemplateLabel">&nbsp;</span>
-                <a data-toggle="modal" data-target="#changeTemplate">
+                <a href="#" data-toggle="modal" data-target="#changeTemplate">
```

Comment:
shouldn't this be a button ?

Reply:
![screen shot 2017-11-08 at 11 02 23](https://user-images.githubusercontent.com/12270833/32542896-5446d87e-c474-11e7-8920-54ac2b767cc4.png)
it isn't a button

Reply:
but it works like a button, didn't they say that a link always needed to go to a page or resource?

Reply:
Maybe adding `role="button"` is the best solution here?